docs/images/profile_manager_entity.png.license

```diff
@@ -0,0 +1,6 @@
+SPDX-FileCopyrightText: 2022  Great Software Laboratory Pvt. Ltd 
+
+SPDX-FileCopyrightText: 2022 Open Networking Foundation <info@opennetworking.org>
```

Comment:
just keep GSLAB name. Since this is new file added  by you

Reply:
I have made the changes

Reply:
@thakurajayL, what are your thoughts on this PR? We're your comments addressed? Is this PR ready to merge?